[UX] Can't build new extensions because of the conflicting dependencies of the fnserver

Following the documentation to build the extensions and putting the fn code as the dependency brings a lot of unnecessary dependencies along with their version conflict.

Gopkg.toml looks like:
```toml
[[constraint]]
  name = "github.com/Shopify/sarama"
  version = "1.16.0"

[[constraint]]
  branch = "route-listeners"
  name = "github.com/fnproject/fn"

[[constraint]]
  branch = "master"
  name = "github.com/orcaman/concurrent-map"

```

main.go
```go
// imports and some code omitted

func main () {
	ctx := context.Background()
	srv := server.NewFromEnv(ctx)

	rl := routeListener{}
	rl.kafkaListeners = cmap.New()
	srv.AddRouteListener(rl)

	srv.Start(ctx)
}
```

when trying to build the following extension binary i get the following error:
```
go build
# github.com/denismakogon/fn-mq-extension/vendor/go.opencensus.io/exporter/jaeger/internal/gen-go/jaeger
vendor/go.opencensus.io/exporter/jaeger/internal/gen-go/jaeger/agent.go:79:20: too many arguments in call to oprot.Flush
	have (context.Context)
	want ()
vendor/go.opencensus.io/exporter/jaeger/internal/gen-go/jaeger/agent.go:122:13: too many arguments in call to oprot.Flush
	have (context.Context)
	want ()
vendor/go.opencensus.io/exporter/jaeger/internal/gen-go/jaeger/jaeger.go:2146:13: too many arguments in call to oprot.Flush
	have (context.Context)
	want ()
vendor/go.opencensus.io/exporter/jaeger/internal/gen-go/jaeger/jaeger.go:2163:14: too many arguments in call to oprot.Flush
	have (context.Context)
	want ()
vendor/go.opencensus.io/exporter/jaeger/internal/gen-go/jaeger/jaeger.go:2176:14: too many arguments in call to oprot.Flush
	have (context.Context)
	want ()
vendor/go.opencensus.io/exporter/jaeger/internal/gen-go/jaeger/jaeger.go:2190:23: too many arguments in call to oprot.Flush
	have (context.Context)
	want ()
# github.com/denismakogon/fn-mq-extension/vendor/github.com/fnproject/fn/api/logs/s3
vendor/github.com/fnproject/fn/api/logs/s3/s3.go:206:4: undefined: view.DistributionAggregation
vendor/github.com/fnproject/fn/api/logs/s3/s3.go:226:4: undefined: view.DistributionAggregation
# github.com/denismakogon/fn-mq-extension/vendor/github.com/fnproject/fn/api/agent/drivers/docker
vendor/github.com/fnproject/fn/api/agent/drivers/docker/docker_client.go:130:4: undefined: view.SumAggregation
vendor/github.com/fnproject/fn/api/agent/drivers/docker/docker_client.go:150:4: undefined: view.CountAggregation
vendor/github.com/fnproject/fn/api/agent/drivers/docker/docker_client.go:170:4: undefined: view.CountAggregation
vendor/github.com/fnproject/fn/api/agent/drivers/docker/docker_client.go:190:4: undefined: view.CountAggregation
```

That might be temporary, however, does seem like there is a problem in-between Fn dependencies that makes Fn a faulty dependency itself.

--- a/Gopkg.lock
+++ b/Gopkg.lock
@@ -509,7 +509,6 @@
   version = "v1.1"
 
 [[projects]]
-  branch = "master"
   name = "go.opencensus.io"
   packages = [
     "exporter/jaeger",
@@ -659,6 +658,6 @@
 [solve-meta]
   analyzer-name = "dep"
   analyzer-version = 1
-  inputs-digest = "476e0b453b4947a75639db11c7a092d3157be8e665eb7a7a2633422f737f5369"
+  inputs-digest = "1da8f3b4885673e5d2b431cf72e0a089da506c4ec6da226d6faaa25aeeff958c"
   solver-name = "gps-cdcl"
   solver-version = 1

--- a/Gopkg.toml
+++ b/Gopkg.toml
@@ -72,4 +72,4 @@ ignored = ["github.com/fnproject/fn/cli"]
 
 [[constraint]]
   name = "go.opencensus.io"
-  branch = "master"
+  revision = "f1af72ab88d638dcc20ea6ecf83c98b59b092559"

